### DISCUSSION TITLE
Timeframe Type

### DISCUSSION THREAD

**Discussion by Elitesparkle** [Heroes-Profile/heroesprofile#701] [Ideas]
I'd like to be able to select all 2.55.4 versions with a single click.

![image](https://github.com/Heroes-Profile/heroesprofile/assets/9367799/444695db-fecd-4add-bc69-87d71ea9d696)

I'd rename Minor Patch to Build and add this new filter as Minor Patch.

**Comment by Zemill** [comment]
I added this awhile back
